kayenta-judge/src/main/scala/com/netflix/kayenta/judge/classifiers/metric/BaseMetricClassifier.scala

```diff
@@ -41,6 +41,20 @@ object MetricDirection{
   }
 }
 
+sealed trait NanStrategy
+object NanStrategy {
```

Comment:
The classifiers do not use the NanStrategy directly. Towards this end, should this object be defined in the BaseMeticClassifier or should it be defined elsewhere?

Reply:
I think the whole thing needs to be refactored as proper objects; when we do that, I think we'll end up with the configuration part moving outside both, and called from either as needed.  For now, let's leave it where it is, since it sits nicely next to the other config options.